Resources/Prototypes/_EE/Entities/Clothing/SolAlliance/SAN/Modsuits/CSA-85x.yml

```
@@ -0,0 +1,192 @@
+          name: power-cell-slot-component-slot-name-default
+          startingItem: PowerCellMicroreactor
+          whitelist:
+            components:
+              - PowerCell
+
+- type: entity
+  parent: [ClothingModsuitGauntletsStandard, SolAllianceInfo]
+  id: ClothingModsuitGauntletsSANMarine
+  name: CSA-85x "Mazu" modsuit gauntlets
+  description: A special modular suit gloves that protect wearer from electric shock.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)